libraries/load-themed-styles/src/index.ts

```diff
@@ -251,10 +251,33 @@ function applyThemableStyles(stylesArray: ThemableArray, styleRecord?: IStyleRec
 export function loadTheme(theme: ITheme | undefined): void {
   _themeState.theme = theme;
 
+  try {
+    const { style } = document.body;
+    for (const key in theme) {
+      if (theme.hasOwnProperty(key)) {
+        style.setProperty(`--${key}`, theme[key]);
+      }
+    }
+  } catch (e) {
+    // Log but suppress error
+    console.error(e);
+  }
+
   // reload styles.
   reloadStyles();
 }
 
+/**
+ * Replaces theme tokens with CSS variable references.
+ * @param styles - Raw css text with theme tokens
+ * @returns A css string with theme tokens replaced with css variable references
+ */
+export function replaceTokensWithVariables(styles: string): string {
```

Comment:
where is this code called? just externally?

Reply:
I'm probably going to do all the tooling to leverage this in `heft-sass-plugin` as part of this PR, so this'll sit here for a while.